### DISCUSSION TITLE
Flows and credentials do not persist on render

### DISCUSSION THREAD

**Discussion by bruceb85** [FlowiseAI/Flowise#1745] [General]
I am hosting flowise on render, but my flows and api credentials dont seem to perist. I am not sure what causes everything to dissapear. It may work for a few days and then I come back and nothing is there but empty flowise dashboard.

**Comment by Jaredude** [comment]
To persist, you will need to have [permanent storage](https://docs.flowiseai.com/configuration/deployment/render#persistent-disk) and the env vars pointing to that permanent storage or persist the data to a remote mysql or Postgres DB.
